### DISCUSSION TITLE
PicklingError: Can't pickle <class 'pyod.models.pca.PCA'>: it's not found as pyod.models.pca.PCA

### DISCUSSION THREAD

**Discussion by Piyushlamsoge** [pycaret/pycaret#3246] [Q&A]
I try to build classification model in google colab, But when i going to save the model its shows the same error everytime 

HERE IS MY CODE :
`!pip install pycaret` 

`import pandas as pd
df = pd.read_csv('/content/Iris.csv')
df`

`from pycaret.classification import *
s = setup(data = df , target = 'Species' , preprocess = True , numeric_features = ['SepalLengthCm' , 'SepalWidthCm' , 'PetalLengthCm' , 'PetalWidthCm'] , transformation = True , remove_outliers = True  )`

`ada = create_model('ada')`

`plot_model(ada , plot = 'feature')`

`model = finalize_model(ada)`

`new_data = df.copy()
new_data.drop('Species' , axis = 1 , inplace = True)
new_data`

`predictions = predict_model(model , data = new_data)
predictions`

`save_model(model , 'my_model')`

AND THE ERROR I'M TALKING ABOUT IS HERE:
![pycaret_issue](https://user-images.githubusercontent.com/88150820/212045593-821d9d89-2e35-4cec-9b6a-7d51b8806df5.png)

**Comment by moezali1** [comment]
@Piyushlamsoge Can you please open this as an issue.

**Reply by Piyushlamsoge** [reply]
okkk
